src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -167,11 +167,6 @@ protected void InitializeContent()
                 // Fill the Content buffer
                 string characterSet = WebResponseHelper.GetCharacterSet(BaseResponse);
 
-                if (string.IsNullOrEmpty(characterSet) && ContentHelper.IsJson(contentType))
-                {
-                    characterSet = Encoding.UTF8.HeaderName;
-                }
-
```

Comment:
Please don't mess style changes with code changes. I suggest to move in new PR.

Reply:
The changes are small, I'm mainly moving the code around and removing redundant code.

Reply:
It doesn't matter - we still have to do a full review.

Reply:
Would you be open to fully review this PR once again? Or should I split it in 2?